src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+:root {
+  --link_padding: 20px;
+}
+
+header {
+  position: sticky;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  width: 1200px;
+  margin: 0 auto;
```

Comment:
The header should stretch to the whole width of the page.

Reply:
Noted thank you have fixed it